api/controllers/service.go

```diff
@@ -78,3 +79,16 @@ func (s *ServiceController) ListServices(c *fireball.Context) (fireball.Response
 	return fireball.NewJSONResponse(200, summaries)
 
 }
+
+func (s *ServiceController) UpdateService(c *fireball.Context) (fireball.Response, error) {
```

Comment:
Can you add the swagger endpoint please?

Reply:
✅